apps/services/auth/public-api/test/utils.ts

```diff
@@ -39,3 +40,19 @@ export const sortDelegations = (delegations: DelegationDTO[]) => {
     return a.id === b.id ? 0 : a.id! < b.id! ? -1 : 1
   })
 }
+
+export type NameIdTuple = [name: string, id: string]
+
+export const getFakeNationalId = () =>
+  faker.helpers.replaceSymbolWithNumber('##########')
+
+export const getFakeName = () =>
+  faker.fake('{{name.firstName}} {{name.lastName}}')
+
+export const getFakePerson = (): NameIdTuple => [
+  getFakeName(),
+  getFakeNationalId(),
+]
```

Comment:
These methods are copy/paste from  [genericStubs](https://github.com/island-is/island.is/blob/c16cd3aa40efc06ef05716eb162cc43ef23b2990/apps/services/auth/api/test/stubs/genericStubs.ts)

I wanted to reuse these method, but we discussed that it would be better in some cases just to copy paste between projects.

Where are these fixture methods?

Reply:
I updated these methods to use methods from fixture